test/index.js

```diff
@@ -266,6 +266,8 @@ test('is.date', function (t) {
   var F = function () {};
   F.prototype = new Date();
   t.notOk(is.date(new F()), 'Date subtype is not date');
+  t.ok(is.date.valid(new Date()), 'new Date() is a valid date');
```

Comment:
please add these tests in a subtest, ie, `t.test('is.date.valid', function (st) {`